main.go

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Add this

```
// +build go1.7
```

as documented here -  https://golang.org/pkg/go/build/#hdr-Build_Constraints - should restrict `go get` as well. 

```
- the target operating system, as spelled by runtime.GOOS
- the target architecture, as spelled by runtime.GOARCH
- the compiler being used, either "gc" or "gccgo"
- "cgo", if ctxt.CgoEnabled is true
- "go1.1", from Go version 1.1 onward
- "go1.2", from Go version 1.2 onward
- "go1.3", from Go version 1.3 onward
- "go1.4", from Go version 1.4 onward
- "go1.5", from Go version 1.5 onward
- "go1.6", from Go version 1.6 onward
- "go1.7", from Go version 1.7 onward
- any additional words listed in ctxt.BuildTags
```